### DISCUSSION TITLE
What about AntiContainer support?

### DISCUSSION THREAD

**Discussion by github-userx** [setvisible/DownZemAll#179] [Ideas]
Do anticontainer still work with this fork?

https://www.downthemall.org/anticontainer-with-easy-installation-of-third-party-plugins/

https://github.com/downthemall/anticontainer/

**Comment by setvisible** [comment]
Hi there,

It looks like certain resources on the Web are protected against batch downloading. (with a sort of container that impersonate the resource but is not the resource?). If there's a C++ ou Python library for that, maybe it could be integrated... 

Rem: This project is not a fork. It's independent. This project uses techologies that don't require the Web Browser (Chrome, Firefox...).

Rgds

**Comment by github-userx** [comment]
Message ID: ***@***.***>Hi.

Thanks for the response.

I guess for photos stuff gallery-dl can be used. And I guess for everything that gallery-dl or yt-dlp can’t get, we have to fall back on using VideoDownloadHelper Browser Addom manually or for scraping headless selenium/puppeteer browser with scripts.

Regards

**Comment by setvisible** [comment]
Good to know about gallery-dl, thanks for the info

This app is a user of yt-dlp, so if a video site doesn't work it's probably better to let yt-dlp fix the issue.

The gallery-dl looks very interesting, maybe it worth to use it.

Regards
